pycbc/events/triggers.py

```diff
@@ -297,32 +293,28 @@ def get_inj_param(injfile, param, ifo):
         Parameter to be calculated for the injected signals
     ifo: string
         Standard detector name, ex. 'L1'
+    args: Namespace object returned from ArgumentParser instance
+        Calling code command line options, used for f_lower value
 
     Returns
     -------
     [return value]: NumPy array of floats
         The calculated parameter values
     """
     det = pycbc.detector.Detector(ifo)
-    time_delay = numpy.vectorize(#lambda dec, ra, t :
-                                 det.time_delay_from_earth_center)#(dec, ra, t))
 
     inj = injfile["injections"]
     if param in inj.keys():
         return inj["injections/"+param]
-    inj_param_dict = {
-        "mtotal" : inj['mass1'][:] + inj['mass2'][:],
-        "mchirp" : conversions.mchirp_from_mass1_mass2(inj['mass1'][:],
-                                                     inj['mass2'][:]),
-        "eta" : conversions.eta_from_mass1_mass2(inj['mass1'][:],
-                                                  inj['mass2'][:]),
-        "effective_spin" : conversions.chi_eff(inj['mass1'][:],
-                                               inj['mass2'][:],
-                                               inj['spin1z'][:],
-                                               inj['spin2z'][:]),
-        "end_time_"+ifo[0].lower() :
-            inj['end_time'][:] + time_delay(inj['longitude'][:],
-                                            inj['latitude'][:],
-                                            inj['end_time'][:]),
-    }
-    return inj_param_dict[param]
+
+    if param == "end_time_"+ifo[0].lower():
+        inj_param = inj['end_time'][:] + det.time_delay_from_earth_center(
```

Comment:
same as above: can just immediately return this value rather than defining inj_param and them returning it